test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare module 'test-case-reporting' {
+  /**
+   * A standard logging interface.
+   */
+  export interface Logger {
+    debug(message: string, ...extraInfo: unknown[]): void;
+    warn(message: string, ...extraInfo: unknown[]): void;
+    error(message: string, ...extraInfo: unknown[]): void;
+    info(message: string, ...extraInfo: unknown[]): void;
+  }
+
+  /**
+   * Possible job types.
+   */
+  export type JobType = 'unit' | 'integration';
+
+  /**
+   * Possible test statuses
+   */
+  export type TestStatus = 'pass' | 'fail' | 'skip' | 'error';
+
+  /**
+   * One concrete Travis build
```

Comment:
I would just say `A build on Travis.` and below `A job within a Travis build.`

On another note, I see the models here are connected the way the relational database connects them. That is, a test run has a job, and a job has a build. When we think about these models, we're more likely to think of a `Build` as containing a list of `Job`s, while a `Job` includes a list of `TestRuns`. We'll chat about this in our next 1:1

Reply:
Done.